### PR TITLE
Update changelog.txt

### PR DESCRIPTION
**Pull request by trwalke**

### CODE REVIEW

File: changelog.txt [hunk 1]
```diff
@@ -1,3 +1,7 @@
+Version 5.0-preview
```
Comment (MarkZuber) [inline changelog.txt]: nit: 5.0.0-preview to be exactly specific...

**Review by MarkZuber — APPROVED** [review]

File: changelog.txt (hunk 1 above)
Comment (trwalke) [reply changelog.txt]: true

File: changelog.txt [hunk 2]
```diff
@@ -1,3 +1,7 @@
+Version 5.0.0-preview
+==============
+- **Enhancements includes support for MSALv3 Cache (which is also or will soon be supported by other libraries such as Python and Java, more detail here https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/wiki/MSAL.NET-3-released) as well as enabling users to bring their own browser as part of the Auth Code retrieval.
```
Comment (jennyf19) [inline changelog.txt:3]: i would use this: [https://aka.ms/msal-net-3x](https://aka.ms/msal-net-3x)

Comment (jennyf19) [inline changelog.txt:3]: >s [](start = 24, length = 1)

i think it should be "Enhancements include" without the 's' but i'm not sure..have to say it out loud.

Comment (jennyf19) [inline changelog.txt:3]: >own browser [](start = 291, length = 11)

you can reference this issue if you want: [https://github.com/AzureAD/azure-activedirectory-library-for-dotnet/issues/1521](https://github.com/AzureAD/azure-activedirectory-library-for-dotnet/issues/1521) for web browser

Comment (trwalke) [reply changelog.txt:3]: cool, ill update

Comment (trwalke) [reply changelog.txt:3]: thanks

Comment (trwalke) [reply changelog.txt:3]: you are right. no s

**Pull request merged**